src/intro.md

```diff
@@ -10,12 +10,31 @@ This is the official “LinuxBoot Book” for the LinuxBoot project. The book:
 
 ## What is LinuxBoot?
 
-LinuxBoot is a project that aims to replace specific [firmware](./glossary.md)
-functionality with a Linux kernel and runtime. Over the years this project has
-grown to include various initiatives with the overarching goal of moving from
-obscure, complex firmware to simpler, open source firmware.
+LinuxBoot is based on the idea of replacing proprietary or corporate-driven
+late-stage boot [firmware](./glossary.md) with the Linux kernel and a
+community-based userland. That idea has grown into a project that over the
+years includes various initiatives with the overarching goal of moving from
+obscure and complex firmware to simpler and open source firmware.
 
-The goal of LinuxBoot is to reduce the role of firmware to a small,
+The project consists of the [linuxboot](https://github.com/linuxboot/linuxboot)
```

Comment:
Suggestion: "The project provides a reference build system that outputs..." and "build system" be the link.
We have many more things, utilities to work with platform features, and also the Heads project as sort of a member / another implementation of LinuxBoot. WDYT?